sound/soc/sof/topology.c

```diff
@@ -476,8 +476,16 @@ struct sof_topology_token {
 	void (*load_token)(void *elem, void *object, u32 offset, u32 size);
 	u32 offset;
 	u32 size;
+	u32 (*get_token)(void *elem);
 };
 
+static u32 get_token_u32(void *elem)
+{
+	struct snd_soc_tplg_vendor_value_elem *velem = elem;
+
+	return le32_to_cpu(velem->value);
+}
+
```

Comment:
same here, could we rename this 'find_token_xyz' to avoid the get/load conflict?

Reply:
The load_token and get_token are more intuitive and representative of what the get/load actually mean, I think. But if you absolutely prefer keeping the get and calling the new one find I'll change it.

Reply:
Load and get sound exactly the same to me. I really don't see any meaning here.

I am in favor of whatever introduces the smallest number of lines changed.

Reply:
done!